src/http/data_loader.rs

```diff
@@ -16,10 +17,30 @@ use crate::json::JsonLike;
 pub struct HttpDataLoader {
   pub client: Arc<dyn HttpClient>,
   pub batched: Option<GroupBy>,
+  #[allow(clippy::type_complexity)]
+  pub get_body_value: Arc<dyn Fn(HashMap<String, Vec<&ConstValue>>, &Cow<str>) -> ConstValue + Send + Sync>,
```

Comment:
it'd better to not supress clippy here but actually move part of the definition to separate type alias and reuse it